0-getting-started/install/centos.md

```diff
@@ -46,7 +46,7 @@ package. Installing these dependencies from the EPEL repositories will
 allow RethinkDB to build more quickly:
 
 ```bash
-sudo rpm -Uvh http://download.fedoraproject.org/pub/epel/6/x86_64/epel-release-6-8.noarch.rpm
```

Comment:
Ah I didn't realize that. Good catch. I'm going to revert that link back to http for now.